studio/pages/account/tokens.tsx

```diff
@@ -20,7 +20,7 @@ const UserAccessTokens: NextPageWithLayout = () => {
       <div className="flex items-center justify-between">
         <FormHeader
           title="Access Tokens"
-          description="Personal access tokens can be used with our management API or the Supabase CLI"
+          description="Personal access tokens can be used with our management API or the Supabase CLI. They inherit the same scopes as your actions on the dashboard."
```

Comment:
> They inherit the same scopes as your actions on the dashboard

What are you trying to say? this sounds vague

Reply:
If its "The personal access tokens will have the same permissions you have" can we not just say that.

Reply:
Discussed offline. Will be making this into an alert box so we have more space to explain in detail.

Reply:
It's not that bad though, i've approved anyway. keep things moving

Reply:
Ok will merge first and improve later if users find it unclear.